daras_ai_v2/web_widget_embed.py

```
@@ -1,9 +1,186 @@
+            format_chat_entry(
+                role=CHATML_ROLE_USER,
+                content_text=prev_input,
+                input_images=prev_input_images,
+                # input_audio=prev_input_audio,
+                input_documents=prev_input_documents,
+            ),
+            format_chat_entry(
+                role=CHATML_ROLE_ASSISTANT,
+                content_text=prev_output,
+            ),
+        ]
```

Ruff (0.15.14)

[warning] Consider iterable unpacking instead of concatenation

(RUF005)